### DISCUSSION TITLE
Favicon

### DISCUSSION THREAD

**Discussion by ethyl2** [dedoc/scramble#386] [Ideas]
I would really like to be able to add a favicon. Right now, the documentation page looks a bit unfinished/amateur without one. But other than that, I am loving scramble!
![Screenshot 2024-05-20 at 12 12 46 PM](https://github.com/dedoc/scramble/assets/6438167/aa7de6af-9a92-4e63-89ad-8d1e83a6e9a4)

**Comment by romalytvynenko** [comment]
@ethyl2 I believe that for now the best way to solve this is to publish package's view: `php artisan vendor:publish --provider="Dedoc\Scramble\ScrambleServiceProvider" --tag="scramble-views"` Then, in a published view you simply add the favicon as you would on a website.

Sure, you'll need to keep an eye on view changes in subsequent releases, but overall this should do the job.
